### PR TITLE
More EAPI2 stuff

### PR DESCRIPTION
**Pull request by flappyports**
@SoapGentoo please review
@robbat2 This include a drop of old EAPI ebuilds from mail-filter/dcc

### CODE REVIEW

**Comment by flappyports** [comment]
@robbat2 and bglibs

**Comment by robbat2** [comment]
+1 on my pkgs here, presume there is another bglibs commit coming.

File: app-misc/logserial/logserial-0.4.2.ebuild [hunk 1]
```diff
@@ -16,16 +16,14 @@ IUSE=""
...
-src_prepare() {
-	epatch "${FILESDIR}"/${P}-gentoo.patch
-}
+PATCHES=( "${FILESDIR}/${P}-gentoo.patch" )
 
 src_compile() {
 	tc-export CC
-	emake || die "emake failed"
+	emake
 }
```
Comment (SoapGentoo) [inline app-misc/logserial/logserial-0.4.2.ebuild]: rename this function to `src_configure` and nix `emake`

File: app-misc/logserial/logserial-0.4.2.ebuild [hunk 2]
```diff
@@ -16,16 +16,14 @@ IUSE=""
...
 
 src_install() {
-	dobin logserial || die "dobin failed"
+	dobin logserial
 	dodoc CHANGELOG README
```
Comment (SoapGentoo) [inline app-misc/logserial/logserial-0.4.2.ebuild]: could also make this `einstalldocs`, your choice

File: app-text/pspresent/pspresent-1.3-r1.ebuild [hunk 3]
```diff
@@ -22,6 +22,7 @@ DEPEND="${RDEPEND}
 	>=sys-apps/sed-4"
```
Comment (SoapGentoo) [inline app-text/pspresent/pspresent-1.3-r1.ebuild]: I don't think this sed is necessary anymore

File: app-i18n/scim-wnn/scim-wnn-1.0.0-r1.ebuild [hunk 4]
```diff
@@ -21,23 +19,20 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig
 	>=sys-apps/sed-4"
```
Comment (SoapGentoo) [inline app-i18n/scim-wnn/scim-wnn-1.0.0-r1.ebuild]: same here, not necessary anymore

File: dev-vcs/rsvndump/rsvndump-0.6.ebuild [hunk 5]
```diff
@@ -22,14 +20,10 @@ DEPEND="${RDEPEND}
 	doc? ( app-text/xmlto
 		>=app-text/asciidoc-8.4 )"
 
+DOCS=( AUTHORS ChangeLog NEWS README THANKS )
```
Comment (SoapGentoo) [inline dev-vcs/rsvndump/rsvndump-0.6.ebuild]: sure this is needed?

File: games-misc/xcowsay/xcowsay-1.3.ebuild [hunk 6]
```diff
@@ -28,7 +28,7 @@ src_configure() {
 }
 
 src_install() {
-	emake DESTDIR="${D}" install || die "emake failed"
+	default
 	if ! use fortune; then
 		rm -f "${D}"/usr/bin/xcowfortune
```
Comment (SoapGentoo) [inline games-misc/xcowsay/xcowsay-1.3.ebuild]: `rm -f "${ED}"/usr/bin/xcowfortune || die`

File: media-gfx/pqstego/pqstego-0.0.1.ebuild [hunk 7]
```diff
@@ -19,8 +19,3 @@ src_configure() {
 	econf \
 		--disable-dependency-tracking
 }
```
Comment (SoapGentoo) [inline media-gfx/pqstego/pqstego-0.0.1.ebuild]: can nix this `src_configure` too

File: media-libs/dssi/dssi-1.1.1-r1.ebuild [hunk 8]
```diff
@@ -24,6 +24,7 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 
 src_prepare() {
+	eapply_user
```
Comment (SoapGentoo) [inline media-libs/dssi/dssi-1.1.1-r1.ebuild]: what happened to `default` as you usually do and should?

File: media-libs/dssi/dssi-1.1.1-r1.ebuild [hunk 9]
```diff
@@ -33,13 +34,8 @@ src_prepare() {
...
-	econf \
-		--disable-dependency-tracking
-}
-
 src_install() {
-	emake DESTDIR="${D}" install || die
-	dodoc README doc/TODO doc/*.txt
+	DOCS=( README doc/TODO doc/*.txt )
+	default
 	find "${D}" -name '*.la' -delete
```
Comment (SoapGentoo) [inline media-libs/dssi/dssi-1.1.1-r1.ebuild]: missing `|| die`

File: media-libs/lastfmlib/lastfmlib-0.4.0.ebuild [hunk 10]
```diff
@@ -18,15 +18,13 @@ DEPEND="${RDEPEND}
...
 		$(use_enable debug) \
 		$(use_enable syslog logging) \
 		--disable-unittests
 }
 
 src_install() {
-	emake DESTDIR="${D}" install || die
-	dodoc AUTHORS ChangeLog
+	default
 	find "${D}"/usr -name '*.la' -delete
```
Comment (SoapGentoo) [inline media-libs/lastfmlib/lastfmlib-0.4.0.ebuild]: `find "${D}" -name '*.la' -delete || die`

File: media-libs/libpqstego/libpqstego-0.0.1.ebuild [hunk 11]
```diff
@@ -16,12 +16,10 @@ DEPEND="sci-libs/gsl[cblas-external]"
...
 	econf \
-		--disable-dependency-tracking \
 		--disable-static
 }
 
 src_install() {
-	emake DESTDIR="${D}" install || die
-	dodoc AUTHORS CHANGES README TODO
+	default
 	find "${D}" -name '*.la' -delete
```
Comment (SoapGentoo) [inline media-libs/libpqstego/libpqstego-0.0.1.ebuild]: `find "${D}" -name '*.la' -delete || die`

File: media-sound/cvoicecontrol/cvoicecontrol-0.9_alpha-r1.ebuild [hunk 12]
```diff
@@ -17,15 +16,16 @@ IUSE=""
 
 S=${WORKDIR}/${MY_P}
 
+PATCHES=( "${FILESDIR}/${P}-gentoo-2.patch" )
+
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-gentoo-2.patch
+	eapply_user
```
Comment (SoapGentoo) [inline media-sound/cvoicecontrol/cvoicecontrol-0.9_alpha-r1.ebuild]: `default`?

File: media-sound/cvoicecontrol/cvoicecontrol-0.9_alpha-r1.ebuild [hunk 13]
```diff
@@ -17,15 +16,16 @@ IUSE=""
...
 	sed -i -e "s/install-data-am: install-data-local/install-data-am:/" Makefile.in
 	# Handle documentation with dohtml instead.
 	sed -i -e "s:SUBDIRS = docs:#SUBDIRS = docs:" cvoicecontrol/Makefile.in
```
Comment (SoapGentoo) [inline media-sound/cvoicecontrol/cvoicecontrol-0.9_alpha-r1.ebuild]: both seds are missing `|| die`

File: net-p2p/syrep/syrep-0.9.ebuild [hunk 14]
```diff
@@ -33,8 +34,7 @@ src_configure() {
 }
 
 src_install() {
-	emake DESTDIR="${D}" install || die
-	cd doc
-	dodoc README *.txt
-	use doc && dohtml *.html *.css
+	DOCS=( doc/README doc/*.txt )
+	use doc && HTML_DOCS=( doc/*.html doc/*.css )
```
Comment (SoapGentoo) [inline net-p2p/syrep/syrep-0.9.ebuild]: `use doc && HTML_DOCS=( doc/*.{css,html} )`

File: net-misc/gofish/gofish-1.2.ebuild [hunk 15]
```diff
@@ -19,6 +19,8 @@ RDEPEND=""
 
 S=${WORKDIR}/${PN}
 
+DOCS=( AUTHORS ChangeLog Configure_GoFish README TODO )
```
Comment (SoapGentoo) [inline net-misc/gofish/gofish-1.2.ebuild]: dont define this, just do `dodoc Configure_GoFish` after `default` in `src_install`

File: net-libs/libosip/libosip-3.6.0.ebuild [hunk 16]
```diff
@@ -19,17 +20,14 @@ IUSE="test"
 
 S=${WORKDIR}/${MY_P}
 
+PATCHES=( "${FILESDIR}/${PN}-3.3.0-out-source-build.patch" )
+
 src_prepare() {
-	epatch "${FILESDIR}/${PN}-3.3.0-out-source-build.patch"
+	eapply_user
```
Comment (SoapGentoo) [inline net-libs/libosip/libosip-3.6.0.ebuild]: `default`?

File: net-libs/libeXosip/libeXosip-4.0.0.ebuild [hunk 17]
```diff
@@ -1,24 +1,29 @@
...
 LICENSE="GPL-2"
-IUSE="+srv ssl"
+IUSE="libressl +srv ssl"
 
 DEPEND=">=net-libs/libosip-4.0.0:=
-	ssl? ( dev-libs/openssl )"
+	ssl? (
+		libressl? ( dev-libs/libressl:0= )
+		!libressl? ( dev-libs/openssl:0= )
+	)
```
Comment (SoapGentoo) [inline net-libs/libeXosip/libeXosip-4.0.0.ebuild]: you tested this? nice 👍

File: net-im/librvp/librvp-0.9.7.ebuild [hunk 18]
```diff
@@ -22,8 +22,3 @@ src_configure() {
 	econf --with-gaim-plugin-dir=/usr/$(get_libdir)/pidgin \
```
Comment (SoapGentoo) [inline net-im/librvp/librvp-0.9.7.ebuild]: `"${EPREFIX}"` in front of paths

File: media-sound/substract_wave/substract_wave-0.3-r1.ebuild [hunk 19]
```diff
@@ -13,17 +13,14 @@ SLOT="0"
...
+PATCHES=(
+	"${FILESDIR}/${P}-Makefile.patch"
+	"${FILESDIR}/${P}-overflow.patch"
+)
+
+DOCS=( CHANGES mono-stereo.txt README )
 
 src_compile() {
 	tc-export CC
 	emake || die "emake failed"
```
Comment (SoapGentoo) [inline media-sound/substract_wave/substract_wave-0.3-r1.ebuild]: nix this line, and rename to `src_configure`

File: net-libs/libeXosip/libeXosip-4.0.0.ebuild (hunk 17 above)
Comment (flappyports) [reply net-libs/libeXosip/libeXosip-4.0.0.ebuild]: yes, tested on my libressl machine

**Pull request closed without merging**